nuxeo-arender-rendition-apb/Dockerfile

```diff
@@ -0,0 +1,66 @@
+IHByb3BlcnRpZXMKICAgIC0gbmFtZTogYXJlbmRlcl9yZW5kaXRpb25faW1hZ2UKICAgICAgdHlw\
+ZTogc3RyaW5nCiAgICAgIHRpdGxlOiBBUmVuZGVyIHJlbmRpdGlvbiBpbWFnZQogICAgICBkZWZh\
+dWx0OiBkb2NrZXItcmVnaXN0cnkuZGVmYXVsdC5zdmM6NTAwMC9jb21tb24taW5mcmEvYXJlbmRl\
+ci1yZW5kaXRpb24KICAgICAgZGlzcGxheV9ncm91cDogUHJvamVjdCBwcm9wZXJ0aWVzCiAgICAt\
+IG5hbWU6IGFyZW5kZXJfaW1hZ2VfdmVyc2lvbgogICAgICB0eXBlOiBzdHJpbmcKICAgICAgdGl0\
+bGU6IEltYWdlIHZlcnNpb24KICAgICAgZGVmYXVsdDogIjEuMC4xIgogICAgICBkaXNwbGF5X2dy\
+b3VwOiBQcm9qZWN0IHByb3BlcnRpZXMKCiAgICAtIG5hbWU6IHByZXZpZXdlcl9pbWFnZV9wdWxs\
+UG9saWN5CiAgICAgIGRlZmF1bHQ6ICdJZk5vdFByZXNlbnQnCiAgICAgIHR5cGU6IGVudW0KICAg\
+ICAgZW51bTogWydJZk5vdFByZXNlbnQnLCAnQWx3YXlzJywgJ05ldmVyJ10KICAgICAgdGl0bGU6\
+IEFSZW5kZXIgUHJldmlld2VyIGltYWdlIHB1bGwgcG9saWN5CiAgICAgIGRpc3BsYXlfZ3JvdXA6\
+IENvbnRhaW5lciBTcGVjcwogICAgLSBuYW1lOiBwcmV2aWV3ZXJfcG9kQW5ub3RhdGlvbnMKICAg\
+ICAgZGVmYXVsdDogJ3t9JwogICAgICB0eXBlOiBzdHJpbmcKICAgICAgdGl0bGU6IEFubm90YXRp\
+b25zIHRvIGJlIGFkZGVkIHRvIEFSZW5kZXIgUHJldmlld2VyIHBvZHMKICAgICAgZGlzcGxheV90\
+eXBlOiB0ZXh0YXJlYQogICAgICBkaXNwbGF5X2dyb3VwOiBDb250YWluZXIgU3BlY3MKICAgIC0g\
+bmFtZTogcHJldmlld2VyX3NlY3VyaXR5Q29udGV4dAogICAgICBkZWZhdWx0OiAne30nCiAgICAg\
+IHR5cGU6IHN0cmluZwogICAgICB0aXRsZTogUHJldmlld2VyIFNlY3VyaXR5IGNvbnRleHQgZm9y\
+IHRoZSBwb2QKICAgICAgZGlzcGxheV90eXBlOiB0ZXh0YXJlYQogICAgICBkaXNwbGF5X2dyb3Vw\
+OiBDb250YWluZXIgU3BlY3MKICAgIC0gbmFtZTogcHJldmlld2VyX3Jlc291cmNlcwogICAgICBk\
+ZWZhdWx0OiAneyJsaW1pdHMiOiB7ImNwdSI6ICIyIiwibWVtb3J5IjogIjJHaSJ9LCJyZXF1ZXN0\
+cyI6IHsiY3B1IjogIjEiLCJtZW1vcnkiOiAiMUdpIn19JwogICAgICB0eXBlOiBzdHJpbmcKICAg\
+ICAgdGl0bGU6IFByZXZpZXdlciBQb2QgcmVzb3VyY2UgcmVxdWVzdHMgYW5kIGxpbWl0cwogICAg\
+ICBkaXNwbGF5X3R5cGU6IHRleHRhcmVhCiAgICAgIGRpc3BsYXlfZ3JvdXA6IENvbnRhaW5lciBT\
+cGVjcwoKICAgIC0gbmFtZTogcmVuZGl0aW9uX2ltYWdlX3B1bGxQb2xpY3kKICAgICAgZGVmYXVs\
+dDogJ0lmTm90UHJlc2VudCcKICAgICAgdHlwZTogZW51bQogICAgICBlbnVtOiBbJ0lmTm90UHJl\
+c2VudCcsICdBbHdheXMnLCAnTmV2ZXInXQogICAgICB0aXRsZTogQVJlbmRlciBSZW5kaXRpb24g\
+aW1hZ2UgcHVsbCBwb2xpY3kKICAgICAgZGlzcGxheV9ncm91cDogQ29udGFpbmVyIFNwZWNzCiAg\
+ICAtIG5hbWU6IHJlbmRpdGlvbl9wb2RBbm5vdGF0aW9ucwogICAgICBkZWZhdWx0OiAne30nCiAg\
+ICAgIHR5cGU6IHN0cmluZwogICAgICB0aXRsZTogQW5ub3RhdGlvbnMgdG8gYmUgYWRkZWQgdG8g\
+QVJlbmRlciBSZW5kaXRpb24gcG9kcwogICAgICBkaXNwbGF5X3R5cGU6IHRleHRhcmVhCiAgICAg\
+IGRpc3BsYXlfZ3JvdXA6IENvbnRhaW5lciBTcGVjcwogICAgLSBuYW1lOiByZW5kaXRpb25fc2Vj\
+dXJpdHlDb250ZXh0CiAgICAgIGRlZmF1bHQ6ICd7fScKICAgICAgdHlwZTogc3RyaW5nCiAgICAg\
+IHRpdGxlOiBTZWN1cml0eSBjb250ZXh0IGZvciB0aGUgUmVuZGl0aW9uIHBvZAogICAgICBkaXNw\
+bGF5X3R5cGU6IHRleHRhcmVhCiAgICAgIGRpc3BsYXlfZ3JvdXA6IENvbnRhaW5lciBTcGVjcwog\
+ICAgLSBuYW1lOiByZW5kaXRpb25fcmVzb3VyY2VzCiAgICAgIGRlZmF1bHQ6ICd7ImxpbWl0cyI6\
+IHsiY3B1IjogIjIiLCJtZW1vcnkiOiAiMkdpIn0sInJlcXVlc3RzIjogeyJjcHUiOiAiMSIsIm1l\
+bW9yeSI6ICIxR2kifX0nCiAgICAgIHR5cGU6IHN0cmluZwogICAgICB0aXRsZTogUmVuZGl0aW9u\
+IFBvZCByZXNvdXJjZSByZXF1ZXN0cyBhbmQgbGltaXRzCiAgICAgIGRpc3BsYXlfdHlwZTogdGV4\
+dGFyZWEKICAgICAgZGlzcGxheV9ncm91cDogQ29udGFpbmVyIFNwZWNzCg=="
```

Comment:
Did you generate that with `apb prepare` or is this just a copy/paste. If it's a copy paster I prefer you put an empty string so that we don't think the `apb prepare` is done.+